### DISCUSSION TITLE
Using Blackhole to route audio from a zoom, google chrome or skype to head phones and Google Chrome input

### DISCUSSION THREAD

**Discussion by nrlevasseur** [ExistentialAudio/BlackHole#668] [Q&A]
I am trying to use google's speech to text to send spoken words from a Zoom session, Skype session, or youtube video to both the headphones and to a google doc input.  I have reviewed the examples and the documentation extensively, but I don't understand which method I should use.  

I don't understand how the aggregate output could provide output to both the headphones and the google Chrome when Chrome is normally using the microphone ... I can hear the speech on my headphones but nothing appears in the google doc until I unplug my headphones and the speech plays through the speakers and is picked up by the mac microphone.

I am happy to provide extensive documentation when I have it working as well as helping support your efforts.  I'm an avid foreign language learner and I can't always pick up the spoken language but the "written" helps when it appears on the screen. A little crutch but so be it.  Thanks for reading this cryptic question.

**Comment by fvogel** [comment]
The following video might help, if I understood your use case well:
https://www.youtube.com/watch?v=sT7YKMTdnX4

"multi output device" and "aggregate device" is probably what you would look for. And playing a bit with different configuration and routing could help.

Edit: I read now that you went down the "aggregate" route already... Then my comment will not help much, sorry.
